cloudwatch-logs-to-logstash/lambda/index.js

```diff
@@ -0,0 +1,46 @@
+const Logstash = require('./Logstash');
+const zlib = require('zlib');
+
+exports.handler = function(event, context) {
+    const payload = new Buffer(event.awslogs.data, 'base64');
+
+    zlib.gunzip(payload, function(e, result) {
+        if (e) {
+            context.fail(e);
+        } else {
+            result = JSON.parse(result.toString('utf8'));
+
+            const logs = result.logEvents;
+            const records = [];
+
+            logs.forEach(function(log) {
+                const messageParts = log.message.split('\t');
+
+                if (! isNaN(Date.parse(messageParts[0]))) {
```

Comment:
Filtering out log lines that were generated in the normal lifecycle of a lambda execution, e.g. duration, memory use. Log lines with timestamps were created explicitly from a lambda calling `console.log()`.

Added comment.